### PR TITLE
Bump version go to 1.19

### LINKED ISSUE #1041: Bump Go minimum to 1.19
Following the [SDK](https://github.com/cosmos/cosmos-sdk/pull/13397)

### PR DESCRIPTION
**Pull request by GNaD13**
Close #1041

### CODE REVIEW

**Comment by alpe** [comment]
Thanks for the PR! This was quicker than planned. Bumping to 1.19 means chains importing the module would also have to upgrade. 
For Context: https://github.com/cosmos/cosmos-sdk/issues/13373#issuecomment-1256801245

**Comment by GNaD13** [comment]
> Thanks for the PR! This was quicker than planned. Bumping to 1.19 means chains importing the module would also have to upgrade. For Context: [cosmos/cosmos-sdk#13373 (comment)](https://github.com/cosmos/cosmos-sdk/issues/13373#issuecomment-1256801245)

Got it. Thanks for your review!!

**Comment by alpe** [comment]
Let's do the upgrade in v0.31 which is getting closer

**Comment by alpe** [comment]
@GNaD13 can you open the PR again, please? We want this for v0.30 after the recent issue on a mainnet

File: CHANGELOG.md [hunk 1]
```diff
@@ -11,7 +11,7 @@
 - Allow AccessConfig to use a list of addresses instead of just a single address [\#945](https://github.com/CosmWasm/wasmd/issues/945)
 - Make contract addresses predictable \("deterministic"\) [\#942](https://github.com/CosmWasm/wasmd/issues/942)
 - Add query for the total supply of a coin [\#903](https://github.com/CosmWasm/wasmd/pull/903) ([larry0x](https://github.com/larry0x))
-- Upgrade go to v1.18 [\#866]https://github.com/CosmWasm/wasmd/pull/866/) ([faddat](https://github.com/faddat))
+- Upgrade go to v1.19 [\#866]https://github.com/CosmWasm/wasmd/pull/866/) ([faddat](https://github.com/faddat))
```
Comment (alpe) [inline CHANGELOG.md:14]: Please add a new line on top

File: README.md [hunk 2]
```diff
@@ -13,7 +13,7 @@ This code was forked from the `cosmos/gaia` repository as a basis and then we ad
 many gaia-specific files. However, the `wasmd` binary should function just like `gaiad` except for the
 addition of the `x/wasm` module.
 
-**Note**: Requires [Go 1.18+](https://golang.org/dl/)
+**Note**: Requires [Go 1.19+](https://golang.org/dl/)
```
Comment (alpe) [inline README.md:16]: 👍

File: go.mod [hunk 3]
```diff
@@ -1,6 +1,6 @@
 module github.com/CosmWasm/wasmd
 
-go 1.18
+go 1.19
```
Comment (alpe) [inline go.mod:3]: 👍

**Comment by GNaD13** [comment]
> @GNaD13 can you open the PR again, please? We want this for v0.30 after the recent issue on a mainnet

Got it :3

File: CHANGELOG.md (hunk 1 above)
Comment (GNaD13) [reply CHANGELOG.md]: I have resolved this. Tysm :3

**Comment by alpe** [comment]
Thanks a lot! 👍

**Pull request merged**